career-link-webapi/appsettings.json

```diff
@@ -1,6 +1,7 @@
 {
+
   "ConnectionStrings": {
-    "DefaultConnection": "data source=localhost;initial catalog=CareerLinkDb;trusted_connection=true;TrustServerCertificate=True"
+    "DefaultConnection": "Server=localhost\\SQLEXPRESS;Database=master;Trusted_Connection=True;TrustServerCertificate=True;"
```

Comment:
refactor your connection string , you should have a database name CareerLinkDb, not changing a new database